lib/models/node.js

```diff
@@ -194,6 +204,31 @@ function NodeModelFactory (Model, waterline, _, Promise, Constants, configuratio
                 return next();
             }
             return next();
+        },
+
+        publishEvent: function(node, next) {
+            return waterline.obms.find({node: node.id})
+            .then(function(obms) {
+                if(node.sku && _.isEmpty(obms)) {
+                    return eventsProtocol.publishNodeAlert(node.id,
+                        { nodeId : node.id,
+                          nodeType: node.type,
+                          state: "identified" });
+                } else if (node.sku && !_.isEmpty(obms)) {
+                    return eventsProtocol.publishNodeAlert(node.id,
+                        { nodeId : node.id,
+                          nodeType: node.type,
+                          state: "managed" });
+                }
+            }).then(function() {
+                next();
+            }).catch(function(err) {
+                next(err);
+            });
+        },
+
+        afterUpdate: function (node, next) {
+            this.publishEvent(node, next);
```

Comment:
What if only the node's name or identifier or other fields (these not either sku or obm) are changed? I think there is no need to publish event is such cases.